src/components/nav/index.js

```diff
@@ -53,14 +53,14 @@ const Nav = () => {
     async function checkRoomForPossibleInvite (room) {
       // Types of spaces for which we want to count invites for
       const context = config.medienhaus?.context ? config.medienhaus?.context.concat('context') : ['context']
-      const content = config.medienhaus?.content ? Object.keys(config.medienhaus?.content).concat('content') : ['content']
-      const typesOfSpaces = context.concat(content)
+      const item = config.medienhaus?.item ? Object.keys(config.medienhaus?.item).concat('item') : ['item']
+      const typesOfSpaces = context.concat(item)
 
       // Ignore if this is not a space
       if (room.getType() !== 'm.space') return
       // Ignore if this is not a student project or a "context"
       const metaEvent = await matrixClient.getStateEvent(room.roomId, 'dev.medienhaus.meta').catch(() => { })
-      if (!metaEvent || !metaEvent.type || !typesOfSpaces.includes(metaEvent.type)) return
+      if (!metaEvent || !metaEvent.template || !typesOfSpaces.includes(metaEvent.template)) return
```

Comment:
So... if we decide to check for `template` here, then let's actually rename the variable `typesOfSpaces` to `templatesOfSpaces`.
But that's only hinting at the actual problem that we discussed 2 days ago: What we actually want to check for is both, first the `type`, and then the `template`, given there are some in the config.

```js
// If the space we're currently checking either doesn't have a metaEvent or not a type given, or has a type given but is neither context nor item, then skip this space (= filter it out)
if (!metaEvent || !metaEvent.type || !['context', 'item'].includes(metaEvent.type)) return;

// Let's get a list of templates from our config we actually allow for first
const contextTemplates = config.medienhaus?.context;
const itemTemplates = config.medienhaus?.item && Object.keys(config.medienhaus?.item);

// If this is a context, and we have a whitelist of context templates from our config, make sure that the space we're looking at is whitelisted; if it's not in the whitelist, skip this space (= filter it out)
if (metaEvent.type === 'context' && contextTemplates && !contextTemplates.includes(metaEvent.template)) return;
// If this is an item, and we have a whitelist of item templates from our config, make sure that the space we're looking at is whitelisted; if it's not in the whitelist, skip this space (= filter it out)
if (metaEvent.type === 'item' && itemTemplates && !itemTemplates.includes(metaEvent.template)) return;
```

Reply:
Can you also check the other changes made in this PR (the ones below) to ensure that they work like this too?